esp/esp/middleware/espauthmiddleware.py

```diff
@@ -31,6 +31,7 @@
   Phone: 617-379-0178
   Email: web-team@learningu.org
 """
+import datetime
```

Comment:
This is probably necessary if we want to turn off `settings.SESSION_EXPIRE_AT_BROWSER_CLOSE` (see #1722).